### PR TITLE
Prepare release 2.6.0

### PR DESCRIPTION
**Pull request by natir**
Changelog:

### Changed

- Replace xz2 to liblzma
- Bump MRSV to 1.65
- Support for multistream bzip2 files by default (#62)

### Fixed

- Update bzip to 0.4.4
- Update bgzip to 0.3
- Update zstd 0.13

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -41,13 +41,13 @@ cfg-if = "1.0"
 thiserror = "1.0"
 bzip2 = { version = "0.4.4", optional = true }
 flate2 = { version = "1.0", optional = true }
-liblzma = { version = "0.3.1", optional = true }
-bgzip = { version = "0.2.2", optional = true }
-zstd = { version = "0.12.2", optional = true }
+xz2 = { version = "0.1", optional = true }
+bgzip = { version = "0.3", optional = true }
+zstd = { version = "0.13", optional = true }
```
Comment (luizirber) [inline Cargo.toml]: We don't want to revert #64 here =]
```suggestion
bgzip = { version = "0.3", optional = true, default-features = false, features = ["rust_backend"] }
liblzma = { version = "0.3.1", optional = true }
zstd = { version = "0.13", optional = true }
```

Also `bgzip` has new default features (`rayon` and `log`) that we don't quite need, so disabling default features and activating the one we actually need

File: Cargo.toml [hunk 2]
```diff
@@ -41,13 +41,13 @@ cfg-if = "1.0"
...
 
 [dev-dependencies]
 tempfile = "3"
-criterion = "0.3"
+criterion = "0.5"
```
Comment (luizirber) [inline Cargo.toml]: This one brings our MSRV to `1.74`. Maybe we can change the MSRV check from `cargo test` to `cargo build`, then we can use newer dev dependencies without issues.

**Review by luizirber — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (natir) [reply Cargo.toml]: God spot

File: Cargo.toml (hunk 2 above)
Comment (natir) [reply Cargo.toml]: In fact we didn't need criterion 0.5 so we can go back to criterion 0.3

**Review by luizirber — APPROVED** [review]
:tada:

**Pull request merged**